run_scripts/3_fetch_recovered_data.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --profile-cpu)
+            PROFILE_CPU=true
+            CPU_PROFILE_OUTPUT_PATH="$2"
+
+            CPU_PROFILE_ARG="--profile-cpu $CPU_PROFILE_OUTPUT_PATH"
+            shift 2;;
+        --)
+            shift
+            break;;
+        *)
+            break;;
+    esac
+done
+
+if [[ $# -ne 4 ]]; then
+    echo "Usage: ./2_fetch_raw_data.sh [--profile-cpu <cpu-profile-output-path>] <user> <google-cloud-credentials-file-path> <pipeline-configuration-file-path> <data-root>"
```

Comment:
I think this needs updating to apply to this file

Reply:
Updated thanks